src/main/java/uk/gov/mca/beacons/api/services/DeleteBeaconService.java

```diff
@@ -76,6 +76,7 @@ public void delete(DeleteBeaconRequestDTO request) {
       .builder()
       .beaconId(beaconIdToBeDeleted)
       .userId(request.getUserId())
+      .fullName("Account Holder")
```

Comment:
Did this basically off the wireframe, but not sure if it's the best way to handle it 🤔 I guess it means less PII exposed in the back office? 

But should the actual name be saved into the DB but then be displayed as "Account Holder" for `Record history` in the Backoffice?

https://miro.com/app/board/o9J_lZuM9qs=/?moveToWidget=3074457358856366472&cot=14